todoman/model.py

```diff
@@ -587,6 +589,8 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             params.extend(lists)
         if urgent:
             extra_where.append('AND priority = 9')
+        elif priority:
+            extra_where.append('AND priority = ' + priority)
```

Comment:
I think this condition needs to be a bit more complex; if I filter `--priority 5`, I'd want anything 1-5 (eg: less than 5, excluding 0).